pages/docs/concepts/channel.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Channel
+weight: 20
+---
+
+# What is a channel?
+A `channel` is a mechanism that is created by the server for the organization and transmission of messages. Channels can also be defined as a _topic ,queue, routing key, path, or subject_ depending on the protocol being used.
+
+# What is the purpose of a channel?
```

Comment:
```suggestion
# Why do we need channels?
```

Sorry for the confusion Thulie, we're going with a `why` question in this heading instead 😅